### DISCUSSION TITLE
Suggestion: Increase the Conflict indicator

### DISCUSSION THREAD

**Discussion by AuWiMo** [snappysnappydog/Simple-IP-Config#164] [Ideas]
I think it would be better for the tiny yellow indicator that shows up when you accidentally cause a conflict to be larger, a popup, or otherwise more notable so that it is evident that something has gone wrong. It is very easy to miss. It would also be nice for it to have some form of error message. 

Now we know what to look out for, but as new users my co-worker and I had some confusion.

**Comment by KurtisLiggett** [comment]
Thanks for the feedback. I added it to the [project roadmap](https://github.com/KurtisLiggett/Simple-IP-Config/discussions/165) to include in the next release!

**Reply by AuWiMo** [reply]
Thank you!

**Comment by KurtisLiggett** [comment]
Now available with release v2.9.7 - [Download Here](https://github.com/KurtisLiggett/Simple-IP-Config/releases/tag/2.9.7)
